yoti-sdk-api/src/main/java/com/yoti/api/client/docs/session/create/SdkConfig.java

```diff
@@ -275,6 +316,47 @@ public Builder withLocale(String locale) {
             return this;
         }
 
+        /**
+         * Sets the dark mode to be used by the web/native client
+         *
+         * @param darkMode the dark mode, e.g. "ON"
+         * @return the builder
+         */
+        public Builder withDarkMode(String darkMode) {
+            this.darkMode = darkMode;
+            return this;
+        }
+
+        /**
+         * Sets the dark mode to 'ON' to be used by the web/native client
+         *
+         * @return the builder
+         */
+        public Builder withDarkModeOn() {
+            this.darkMode = "ON";
```

Comment:
Can we move these String values into the `DocScanConstants` class please?

Reply:
Done, thank you